packages/@aws-cdk/aws-ecs/test/ec2/integ.default-capacity-provider.ts

```diff
@@ -31,11 +31,12 @@ const cp = new ecs.AsgCapacityProvider(stack, 'EC2CapacityProvider', {
 const cluster = new ecs.Cluster(stack, 'EC2CPCluster', {
   vpc,
   enableFargateCapacityProviders: true,
-  defaultCapacityProviderStrategy: [
-    { capacityProvider: 'FARGATE', base: 1, weight: 1 },
-  ],
 });
 
+cluster.addDefaultCapacityProviderStrategy([
+  { capacityProvider: 'FARGATE', base: 1, weight: 1 },
+]);
+
```

Comment:
Let's add this after the asg capacity provider and add a default capacity provider strategy that includes both Fargate and the ASG provider.